api/app/src/main/kotlin/packit/model/dto/BasicPacketDto.kt

```diff
@@ -0,0 +1,6 @@
+package packit.model.dto
+
+data class BasicPacketDto(
+    val name: String,
+    val id: String,
+)
```

Comment:
Given that the 'Basic' Packet Dto, the Tag Dto and the PacketGroup Dto are all just ids and names, I think there's an argument for having a single PermissionScope model which has name and id, and scope type. It going to make wrangling these things easier in the front end. But I guess I already lost that argument in the database and POST methods, and you can always put some utils in the front end to work around the awkwardness.

Reply:
the naming is a bit weird i think il try rename... but I don't think scope thing will help, just means one more field to handle.... the FE will be easy because it will just map through them all and display name (2 of them will be null thus nothing will show)